lib/tasks/migrator.js

```diff
@@ -64,5 +46,60 @@ module.exports = function runMigrations(context) {
         }
 
         return Promise.reject(error);
-    });
-}
+    }
+};
+
+module.exports.migrate = function runMigrations(context) {
+    const {dir} = context.instance;
+
+    const contentDir = path.join(dir, 'content');
```

Comment:
I don't think this was added in this PR, but shouldn't we programattically get this (i.e. `instance.config.get('paths.contentPath')`)